source/Octopus.Client/OctopusAsyncClient.cs

```diff
@@ -138,9 +137,7 @@ private static async Task<IOctopusAsyncClient> Create(OctopusServerEndpoint serv
             try
             {
                 var rootResource = await client.EstablishSession().ConfigureAwait(false);
-                var spaceRootResource = await LoadSpaceRootResource(client, rootResource);
-                client.RootDocuments = new RootResources(rootResource, spaceRootResource);
-                client.Repository = new OctopusAsyncRepository(client);
+                client.Repository = await OctopusAsyncRepository.Create(client);
```

Comment:
RootDocument is no longer in client
Instead of using the constructor, we are now using the Create static method